### DISCUSSION TITLE
need number calculation method in alert message template

### DISCUSSION THREAD

**Discussion by xux1217** [grafana/grafana#59965] [General]
hi,

My Grafana version: v9.3.1

Need function:
I want to contain a panel link in alert message, pointing to the time range when the alert appeared, there is not existing link to finish it.

so I want to generate it by myself, but I can't use like this statement: 
{{.PanelURL}}&from={{add .StartsAt -300}}&to={{add .StartsAt 300}}, there is not "add" function.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
